README.md

```diff
@@ -1,116 +1,101 @@
+after the initial build changes should be picked up quickly and the development server
+should be snappy to use.
 
-### Local development
+### Running Verification Scripts
 
-When developing locally, you can use the `hugo server` mode to start a local
-webserver that watches for changes on your filesystem and automatically
-refreshes the generated content and page when changes are detected.
+After you have made changes to the website, you should run the `verify` scripts
+to ensure things like spelling and links are valid.
 
-To run this mode, run:
+To run all verification checks:
 
-```
-./scripts/server
+```bash
+./scripts/verify
 ```
 
-This will start a local webserver on `http://localhost:1313` where you can view
-the built site.
+This will automatically run a number of checks against your local environment.
 
-There is also a `./scripts/server-release` command which will fetch all
-versions of the documentation content before running the regular `server`
-script.
+If you want to be thorough, you can run `./scripts/verify-release` to also regenerate API / CLI docs
+before verification, but that check is slower and unlikely to provide any useful insight.
 
-### Building output HTML
+### Building for a Release
 
-If you want to build a directory containing the built HTML, you can run the
-following command:
+On release, all output is placed into the `out/` directory.
 
-```
-./scripts/build
-```
+Building a full release includes re-running API + CLI doc generation for the latest
+version of cert-manager, and then running a next.js `build` followed by `export`. The full
+release process can be run through one script:
 
-This will generate a directory named `public/` that can be easily served by the
-default configuration of most webservers.
+```bash
+./scripts/build-release
+```
 
-There is also a `./scripts/build-release` command which will fetch all
-versions of the documentation content before running the regular `build`
-script.
+If you want to test that the build still works locally, you can run `./scripts/build` to build while
+skipping regeneration of API / CLI docs.
 
-### Running verification scripts
+### API / CLI Documentation Generation
 
-After you have made changes to the website, you should run the `verify` scripts
-to ensure things like spelling and links are valid.
+To generate API / CLI reference docs manually, run:
 
-To run all the verify checks, run:
-
-```
-./scripts/verify
+```bash
+./scripts/gendocs/generate
 ```
 
-This will automatically run a number of checks against your changes.
+Since there are many old versions of cert-manager, none of which change regularly (or at all),
+the website build process does not re-generate documentation for older versions, on the assumption
+that doing so would be a waste of effort.
+
+The solution for achieving this is simple; the generation scripts for older cert-manager versions
+are commented out. To rebuild, uncomment them and then re-comment after.
```

Comment:
To regenerate documentation for older versions...

Reply:
I feel this is clear enough as-is; thanks for the suggestion though!